django_addanother/views.py

```diff
@@ -9,6 +10,9 @@ class PopupMixin(object):
     """Mixin for :class:`~django.views.generic.edit.CreateView` classes that
     handles the case of the view being opened in an *add another* popup window.
     """
+
+    POPUP_ACTION = 'add'
```

Comment:
Not for a middle version increment if it's properly documented on such a small package. We'll also need a paste of the traceback that happens when this package is upgraded without the renaming done in the project, for posterity in the documentation and google-fu treasure hunting action by other nerds like us LOL